sky/server/server.py

```diff
@@ -1841,6 +1855,30 @@ async def kubernetes_pod_ssh_proxy(websocket: fastapi.WebSocket,
         async def websocket_to_ssh():
             try:
                 async for message in websocket.iter_bytes():
+                    if timestamps_supported:
+                        message_type = struct.unpack('!B', message[:1])[0]
+                        if (message_type ==
+                                KubernetesSSHMessageType.REGULAR_DATA):
+                            # Regular data - strip type byte and forward to SSH
+                            message = message[1:]
+                        elif message_type == KubernetesSSHMessageType.PINGPONG:
+                            # PING message - respond with PONG (type 1)
+                            pong_message = struct.pack(
+                                '!B', KubernetesSSHMessageType.PINGPONG.value)
+                            await websocket.send_bytes(pong_message)
```

Comment:
I would assume it's safe to call `send_bytes` from two different coroutines at the same time is safe, but have we tested it?